selectors/selectors.md

```diff
@@ -115,47 +110,15 @@ These components imply or expand into the following things:
 Selector types
 --------------
 
-### Fields Selector
-
-Selects each of a set of path segments.
-
-(If only a single path segment is specified, this can be called a "Path Selector", but it's the same thing.)
-
-
-### Path Selector
-
-Selects a single path segment.
-
-Path selectors are a common pattern of usage, but not a specific feature in the selector spec --
-it's a degenerate case of a Fields Selector which happens to select only a single field.
-
-
-### All Selector
-
-Selects every field or index in a map or list.
-
-
-### Range Selector
-
-Selects a slice or single item of a list of items.
-
-
-### Recursive Selector
-
-Uses another Selector to select things recursively. It also takes a limit on how often this recursion should happen (the limit can be set to "infinity" to keep traversing until there are no matching paths left.
-
-The recursion may also be stopped by a given CID.
-
-
-### CID Selector
-
-There is no CID selector.  Asking for a specific CID doesn't involve any traversal; it's just asking for a node itself.  No selectors are necessary to ask for data by CID.
-
+See [Schema](#schema) section below.
 
 
 Use cases
 ---------
 
+WARN: not all of these examples have been updated recently.
```

Comment:
Similar to above -- IMHO it's better to have no example at all than one that is out of date. If we need to update and want to build of what was there we can always look at git history.